test/integration/job/job_test.go

```diff
@@ -1667,6 +1667,165 @@ func TestIndexedJob(t *testing.T) {
 	validateTerminatedPodsTrackingFinalizerMetric(t, 5)
 }
 
+func TestJobPodReplacementPolicy(t *testing.T) {
+	const initialCompletions int32 = 2
+	var podReplacementPolicy = func(obj batchv1.PodReplacementPolicy) *batchv1.PodReplacementPolicy {
+		return &obj
+	}
+	var completionPolicy = func(obj batchv1.CompletionMode) *batchv1.CompletionMode {
+		return &obj
+	}
+	jobSpecIndexedDefault := &batchv1.JobSpec{
```

Comment:
nit: It is used only in two tests and it rather concise, I would suggest inlining the spec.

Reply:
Applied your suggestion.